core/src/main/resources/hudson/model/Job/configure.jelly

```diff
@@ -30,6 +30,15 @@ THE SOFTWARE.
 
   <l:layout norefresh="true" type="one-column" permission="${it.EXTENDED_READ}" title="${%Config(it.displayName)}">
 
+    <j:choose>
+      <j:when test="${app.hasPermission(it.CONFIGURE)}">
+        <j:set var="displayOnlyMode" value="false" />
+      </j:when>
+      <j:otherwise>
+        <j:set var="displayOnlyMode" value="true" />
```

Comment:
This seems a weird choice of name given that both Extended Read and System Read use the term _Read_. I'm also more familiar with "read only" as a term.

Why was this term chosen?

Reply:
Planning to rename it to read, naming is hard.